unfoldingWord/translationCore
# Missing Verses Tests

#### This pull request addresses:
This PR tests the functionality of the missing verses methods


#### How to test this pull request:
run 
<code>npm test</code> 
and ensure all test are passing


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2556)
